### PR TITLE
576: Updated layout with default color scheme

### PR DESCRIPTION
**Pull request by st-jay**
## Summary of change
- Use default color scheme with updated layout

## Related issues
- https://github.com/supertokens/main-website/issues/576

## Checklist
- [ ] Algolia search needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Sitemap needs to be updated? (If there is a new sub docs project, then yes)
- [ ] Checked for broken links? (Run `cd v2 && MODE=production npx docusaurus build`)
- [ ] Changes required to the demo apps corresponding to the docs?

## Remaining TODOs for this PR
- [ ] Item1
- [ ] Item2

### CODE REVIEW

File: v2/src/components/appInfoForm/index.tsx [hunk 1]
```diff
@@ -237,24 +237,41 @@ export default class AppInfoForm extends React.PureComponent<PropsWithChildren<P
         </span>
     )
 
+    getCurrentFormValuesAsString = () => {
```
Comment (nkshah2) [inline v2/src/components/appInfoForm/index.tsx:240]: What is this function for?

File: v2/src/components/question/index.tsx [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import React, { Children, PropsWithChildren, useState } from "react";
 
+let styles = require("./question.module.css").default;
```
Comment (nkshah2) [inline v2/src/components/question/index.tsx:3]: Cant we just import?

File: v2/thirdpartyemailpassword/quick-setup/backend.mdx [hunk 3]
```diff
@@ -369,6 +369,8 @@ init(
 
 </AppInfoForm>
 
+
+
```
Comment (nkshah2) [inline v2/thirdpartyemailpassword/quick-setup/backend.mdx:373]: Why the extra whitespace?

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: v2/thirdpartyemailpassword/quick-setup/backend.mdx (hunk 3 above)
Comment (st-jay) [reply v2/thirdpartyemailpassword/quick-setup/backend.mdx]: updated

File: v2/src/components/question/index.tsx (hunk 2 above)
Comment (st-jay) [reply v2/src/components/question/index.tsx]: No, using `import` isn't working on this file as it isn't declared as a module

File: v2/src/components/appInfoForm/index.tsx (hunk 1 above)
Comment (st-jay) [reply v2/src/components/appInfoForm/index.tsx:240]: This function is used to generate a string which we display inside the snippet of `AppInfoForm` values ( as shown in below images ). It checks for the fields that the current AppInfoForm instance handles and only returns their values.

Screenshot 2022-03-18 at 12 50 59 PM

Screenshot 2022-03-18 at 12 51 09 PM

**Comment by nkshah2** [comment]
Closing this PR in favour of opening separate PRs for each change to make review easier

**Pull request closed without merging**